src/imagetextalternative.js

```diff
@@ -98,10 +95,14 @@ export default class ImageTextAlternative extends Plugin {
 	 */
 	_createBalloonPanel() {
 		const editor = this.editor;
+		const command = editor.commands.get( 'imageTextAlternative' );
 
 		const panel = new ImageBalloonPanel( editor );
 		const form = new TextAlternativeFormView( editor.locale );
 
+		form.labeledInput.inputView.bind( 'isReadOnly' ).to( command, 'isEnabled', value => !value );
+		form.saveButtonView.bind( 'isEnabled' ).to( command );
```

Comment:
I remember why I didn't do it. Because LinkFormView is bind to two commands `LinkCommand` and `UnlinkCommand`. But image alt use only one command so can have a common TextAlternativeFormView#isEnabled state.